analytics/lib/counts.py

```diff
@@ -316,6 +316,7 @@ def do_increment_logging_stat(
     if stat.frequency == CountStat.DAY:
         end_time = ceiling_to_day(event_time)
     else:  # CountStat.HOUR:
+        assert stat.frequency == CountStat.HOUR
         end_time = ceiling_to_hour(event_time)
 
```

Comment:
I would probably prefer to write this with `elif` and then an `else: raise AssertionError("Unsupported CountStat frequency")` for readability reasons.